src/main/java/org/broadinstitute/hellbender/utils/IndexUtils.java

```diff
@@ -0,0 +1,109 @@
+package org.broadinstitute.hellbender.utils;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.SAMSequenceRecord;
+import htsjdk.tribble.AbstractFeatureReader;
+import htsjdk.tribble.Tribble;
+import htsjdk.tribble.index.Index;
+import htsjdk.tribble.index.IndexFactory;
+import htsjdk.tribble.util.ParsingUtils;
+import htsjdk.tribble.util.TabixUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.IndexFeatureFile;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public final class IndexUtils {
+    private IndexUtils(){}
+
+    private static final Logger logger = LogManager.getLogger(IndexUtils.class);
+
+    /**
+     * Load the index from disk, checking for out of date indexes and old versions
+     * @return an Index, or null if we're unable to load
+     */
+    public static Index loadTribbleIndex(final File featureFile) {
+        Utils.nonNull(featureFile);
+        final File indexFile = Tribble.indexFile(featureFile);
+        if (! indexFile.canRead()) {
+            return null;
+        }
+        logger.debug("Loading Tribble index from disk for file " + featureFile);
+        final Index index = IndexFactory.loadIndex(indexFile.getAbsolutePath());
```

Comment:
`loadIndex()` can throw various kinds of exceptions that you are not catching here, including `TribbleException` and `RuntimeException`. Catch these, and return null when they occur instead of blowing up.